src/main/java/cn/nukkit/Player.java

```diff
@@ -1639,10 +1639,10 @@ public boolean onUpdate(int currentTick) {
         this.lastUpdate = currentTick;
 
         if (!this.isAlive() && this.spawned) {
-            ++this.deadTicks;
```

Comment:
this.tick = this.deadTicks; actually define dead ticks to be ticking server and it's ok.